pkg/plugin/plugin.go

```diff
@@ -184,7 +184,7 @@ func (p *plugin) RecoverEtcdCluster(ctx context.Context, cs *api.OpenShiftManage
 	}
 
 	p.log.Info("running CreateOrUpdate")
-	if err := p.CreateOrUpdate(ctx, cs, true, deployFn); err != nil {
+	if err := p.createOrUpdateExt(ctx, cs, updateTypeNormal, deployFn); err != nil {
```

Comment:
call the Ext version so we dont' backup before recovering from a backup

Reply:
Can we add a comment along those lines?

Reply:
sure